notebooks/notebooks/iclr2022/estimate-flops-katago.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    df_nets.net_size.unique()
</pre><p>Consider assigning this to a variable given it's used again later, would emphasize it's not just a throw-away statement.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Assigned to global variable and changed later usage to reference said global.